test/support/test_case.rb

```diff
@@ -10,9 +10,16 @@ module DEBUGGER__
   class TestCase < Test::Unit::TestCase
     include TestUtils
     include AssertionHelpers
+    RC_FILENAME = File.join(Dir.home, ".rdbgrc")
```

Comment:
Do not modify the `~/.rdbgrc` on a test.

Reply:
It only renames the file while the test started and rename it back when it's finished. So eventually the file isn't changed at all. Without this, existing `~/.rdbgrc` will break the tests.

Reply:
Surprisingly, there are platforms that doesn't have HOME (and Ruby tests run on them).

Please use `Dir.chroot` to manipulate home dir.
(make a temp dir and make home dir under the temp dir. And use chroot)